bin/run-demo.sh

```diff
@@ -31,24 +30,15 @@ fi
 shift $((OPTIND - 1))
 
 docker run \
-    --name ${RUN_ID} \
+    --name ${NAME} \
     --volume ${VOLUME_SOURCE}:/project \
     ${DOCKER_IMAGE} \
     process /project/rector_analyzed_file.php \
     --output-format json \
-    --output-file /project/output.json \
     --config /project/rector.yaml
 
 RECTOR_RUN_EXIT_CODE=$?
 
-if [[ ${RECTOR_RUN_EXIT_CODE} -eq 0 ]]
-then
-    cat ${RUN_DIRECTORY}/output.json
-else
-    docker logs ${RUN_ID}
-fi
-
-docker rm ${RUN_ID} > /dev/null
-rm -rf ${RUN_DIRECTORY}
```

Comment:
This was the security issue. It is gone now :-) no1 wants to run `rm -rf` as sudo